libpod/container_internal.go

```diff
@@ -1910,3 +1925,36 @@ func (c *Container) hasNamespace(namespace spec.LinuxNamespaceType) bool {
 	}
 	return false
 }
+
+// getKVMLabel gets an KVM SELinux type and inserts it into the processlabel
+func getKVMLabel(processLabel string) (string, error) {
+	if processLabel == "" {
+		return "", nil
+	}
+	kvmLabel, _ := selinux.KVMContainerLabels()
+	selinux.ReleaseLabel(kvmLabel)
+	return swapTypes(kvmLabel, processLabel)
+}
+
+// getInitLabel gets an init SELinux type and inserts it into the processlabel
+func getInitLabel(processLabel string) (string, error) {
+	if processLabel == "" {
+		return "", nil
+	}
+	initLabel, _ := selinux.InitContainerLabels()
+	selinux.ReleaseLabel(initLabel)
+	return swapTypes(initLabel, processLabel)
+}
+
+func swapTypes(source, dest string) (string, error) {
```

Comment:
Can we rename it to `swapSELinuxTypes`? `libpod` is a really big package and `swapTypes` sounds very generic.

Reply:
Done